Gemfile.lock

```diff
@@ -419,17 +419,17 @@ GEM
     rspec-rerun (1.1.0)
       rspec (~> 3.0)
     rspec-support (3.12.0)
-    rubocop (1.48.0)
+    rubocop (1.50.2)
```

Comment:
I believe this might be causing an issue in CI, or it's a flakey test - will kick of a rerun to confirm

```
RuboCop::Cop::Lint::ModuleEnforceNotes FFFFFFFFFF

  1) RuboCop::Cop::Lint::ModuleEnforceNotes requires Stability, Reliability and SideEffects to be present when no keys are present
     Failure/Error: let(:config) { RuboCop::Config.new }

     Encoding::UndefinedConversionError:
       U+00A9 from UTF-8 to ASCII-8BIT
     # ./spec/rubocop/cop/lint/module_enforce_notes_spec.rb:8:in `new'
     # ./spec/rubocop/cop/lint/module_enforce_notes_spec.rb:8:in `block (2 levels) in <top (required)>'
     # ./spec/rubocop/cop/lint/module_enforce_notes_spec.rb:7:in `block (2 levels) in <top (required)>'
     # ./spec/rubocop/cop/lint/module_enforce_notes_spec.rb:50:in `block (2 levels) in <top (required)>'

```

Last time I had issues with encoding, it was framework's global ASCII IO code here that caused issues with third party gems

https://github.com/rapid7/metasploit-framework/blob/f6c8181b7f3edc3f29f41a323c2f8eb054619cd5/lib/metasploit/framework/common_engine.rb#L29-L33

Reply:
@adfoster-r7 Ran this again, seems this is still an issue so we will likely need to do some updates to fix this up.

Reply:
PR to update https://github.com/rapid7/metasploit-framework/pull/17950

Reply:
Thanks will review that now!